src/send-email.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const nodemailer = require('nodemailer');
+
+// create reusable transporter object using the default SMTP transport
+let transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'welcome.to.cahms@gmail.com',
+    pass: 'Cahms100'
```

Comment:
We have added this to our env file now. I have changed this password too so it's not on GitHub.